app/src/test/java/com/github/se/orator/model/profile/UserProfileViewModelTest.kt

```diff
@@ -36,7 +40,7 @@ class UserProfileViewModelTest {
   fun setUp() {
     MockitoAnnotations.openMocks(this)
     Dispatchers.setMain(testDispatcher)
-
+    mockContext = mock(Context::class.java)
```

Comment:
You can also use Localcontext.current in your screen directly (or make it a default parameter) without having to mock it and pass it as a parameter

Reply:
Are you sure this is possible ?  I'm not using mockContext as class argument but to retrieve filesDir